tensorflow/lite/kernels/connected_components.cc

```diff
@@ -0,0 +1,428 @@
+// This custom op is a port of the connected_components module from tensorflow_addons.
```

Comment:
Can we also link to the reference, non tf-lite implementation?

Reply:
Done!